.github/workflows/check-backport.yml

```diff
@@ -0,0 +1,50 @@
+name: Stable Backport Check
+on:
+  issue_comment:
+  pull_request_target:
+    types:
+      - labeled
+
+jobs:
+  check-backport:
+    name: Check Backport
+    if: ${{ ( github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot check backport') ) || github.event.label.name == 'stable backport' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify as seen
+        uses: peter-evans/create-or-update-comment@v1.4.5
+        if: ${{ github.event.comment != null }}
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          comment-id: ${{ github.event.comment.id }}
+          reactions: '+1'
+
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          fetch-depth: 0
+
+      - name: Perform test backport
+        run: |
+          git config --global user.name "Jellyfin Bot"
+          git config --global user.email "team@jellyfin.org"
+          CURRENT_BRANCH=$( git symbolic-ref --short HEAD )
+          git checkout master
+          git merge --no-ff ${CURRENT_BRANCH}
+          MERGE_COMMIT_HASH=$( git log -q -1 | head -1 | awk '{ print $2 }' )
+          git fetch --all
+          CURRENT_STABLE=$( git branch -r | grep 'origin/release' | sort -rV | head -1 | awk -F '/' '{ print $NF }' )
+          echo "Current stable branch: ${CURRENT_STABLE}"
+          git checkout -t origin/${CURRENT_STABLE} -b ${CURRENT_STABLE}
+          git pull --rebase
+          git cherry-pick -sx -m1 ${MERGE_COMMIT_HASH} 
+
+      - name: Notify with result
+        uses: peter-evans/create-or-update-comment@v1.4.5
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Check build log:
+            ${{ steps.build.outputs.build-log }}
```

Comment:
The output of this is not defined anywhere. [See this](https://stackoverflow.com/questions/59191913/get-output-of-a-specific-step-in-github-actions) and [this](https://github.com/jellyfin/jellyfin-client-axios/blob/master/.github/workflows/stable-generation.yml#L30-L37)

You can build the URL by concatenating the different variables exposed here: https://docs.github.com/es/actions/reference/environment-variables#default-environment-variables

Reply:
Was fixed after more testing.